### PR TITLE
Put GA in the browser

### PR DESCRIPTION
**Pull request by psoleckimoj**
- Initial replace page track event
- Remove backend page track
- Add GA code

### CODE REVIEW

File: server/views/components/template.njk [hunk 1]
```diff
@@ -12,7 +12,24 @@
...
     <meta name="robots" content="noindex, nofollow, noimageindex">
-    {# Hardcoded value of $govuk-black #}
+    {% block analytics %}
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-152065860-4"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-152065860-4', {
```
Comment (psoleckimoj) [inline server/views/components/template.njk:22]: These will need passing through from config.

File: server/views/pages/category.html [hunk 2]
```diff
@@ -137,20 +137,4 @@ <h2 class="govuk-heading-m">Give us feedback</h2>
     .GOVUKFrontend
     .initAll();
 </script>
-<!--[if gte IE 9]>
```
Comment (psoleckimoj) [inline server/views/pages/category.html:140]: We're on Edge, this is no longer required.

File: server/routes/search.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const { path } = require('ramda');
 const express = require('express');
 
-const createSearchRouter = ({ searchService, analyticsService }) => {
+const createSearchRouter = ({ searchService }) => {
```
Comment (psoleckimoj) [inline server/routes/search.js:4]: event removed as this is logged in GA anyway

Comment (psoleckimoj) [reply server/routes/search.js:4]: It was only originally in as I didn't know about the search part of GA

**Review by davidthomas22 — APPROVED** [review]
LGTM

File: server/app.js [hunk 4]
```diff
@@ -169,6 +168,12 @@ const createApp = ({
   // establishment toggle
   app.use(configureEstablishment());
 
+  app.use((req, res, next) => {
+    res.locals.feedbackId = uuid();
```
Comment (psoleckimoj) [inline server/app.js:172]: Moved back in here as it's not really part of establishment.

**Review by davidthomas22 — DISMISSED** [review]
LGTM

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

**Pull request closed without merging**